code/game/machinery/washing_machine.dm

```diff
@@ -52,15 +52,15 @@
 
 
 	if(crayon)
-		var/_color
+		var/item_color

```

Comment:
What does it matter? In this case especially, it's about the color of the crayon, which is an item.